.ci/configure_containerd_for_kata.sh

```diff
@@ -10,7 +10,7 @@ set -o nounset
 set -o pipefail
 
 if [ "$KATA_BUILD_CC" == "yes" ]; then
```

Comment:
This entire `if` block can be removed as `PREFIX="${PREFIX:-/opt/kata}"`  is assigned next (line 15).